### PR TITLE
chore: update glob-parent

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-2533]

### Description

Upgrade [glob-parent](https://github.com/gulpjs/glob-parent/releases) to latest version to fix [high vulnerability](https://github.com/toptal/picasso/security/dependabot/24)

Previous versions:
5.1.2 - lerna
3.1.0 - storybook, webpack

In version 5.1.2 the vulnerability was already fixed, but since in version 6, there were no API changes we can upgrade to a higher version.

Since the storybook is already in the latest version we need to add `glob-parent` to resolutions

I have tested:
1. `lerna version --no-push` with local change commited
2. `lerna build:package`
3. `yarn start`
4. `yarn test:cypress`

### How to test

- FIXME: Add the steps describing how to verify your changes

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
5. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2533]: https://toptal-core.atlassian.net/browse/FX-2533?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -149,6 +149,7 @@
     "prettierx": "^0.17.0",
     "@storybook/react": "^6.4.19",
     "merge-deep": "^3.0.3",
-    "webpack": "^4.0.0"
+    "webpack": "^4.0.0",
+    "glob-parent": "^6.0.2"
```
Comment (dmaklygin) [inline package.json:153]: 👍

Comment (dmaklygin) [reply package.json:153]: I noticed, that `glob-parent` is also used by `chokidar`

**Review by dmaklygin — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (TomasSlama) [reply package.json:153]: yep, there is some multiple dependency chain, but in the end, it is sub dependency of `webpack`

**Review by ozgurkececioglu — APPROVED** [review]
Tested some pages and commands, looks good 👍

**Review by OndrejTuma — APPROVED** [review]

**Comment by TomasSlama** [comment]
@toptal-bot run build

**Comment by TomasSlama** [comment]
@toptal-bot run visual

**Pull request merged**